WcaOnRails/app/webpacker/components/Panel/Wfc/DuesRedirect.jsx

```diff
@@ -115,6 +124,21 @@ export default function DuesRedirect() {
           </Form>
         </Modal.Content>
       </Modal>
+      <Confirm
+        open={toDeleteId}
+        content="Are you sure you want to delete?"
+        onCancel={() => setOpen(false)}
+        onConfirm={() => {
+          const duesRedirectId = toDeleteId;
+          setToDeleteId(null);
```

Comment:
Setting the toDelete ID to null should only happen within the success callback.
This will also save you from copying the old value into a const.

Reply:
Done